src/Repl.vue

```diff
@@ -40,6 +40,11 @@ export interface Props {
     showErrorText?: string
     monacoOptions?: monaco.editor.IStandaloneEditorConstructionOptions
   }
+  splitPaneOptions?: {
+    CodeTogglerButtonText?: string
+    OutputTogglerButtonText?: string
+  }
+  isEmbedMode?: boolean
```

Comment:
As https://github.com/vuejs/repl/pull/279#pullrequestreview-2278144262 said, embed mode can be implemented without any javascript.